src/tests/ftest/osa/online_reintegration.py

```diff
@@ -60,8 +63,13 @@ def run_online_reintegration_test(self, num_pool, racer=False, server_boot=False
         pool = {}
         exclude_servers = (len(self.hostlist_servers) * 2) - 1
 
-        # Exclude one rank : other than rank 0.
-        rank = random.randint(1, exclude_servers)  # nosec
+        if multiple_ranks is True:
```

Comment:
You could simplify it as:
```suggestion
        if multiple_ranks:
```